src/core/umount.c

```diff
@@ -446,15 +529,19 @@ static int mount_points_list_umount(MountPoint **head, bool *changed, bool log_e
 
                 /* Trying to umount. We don't force here since we rely
                  * on busy NFS and FUSE file systems to return EBUSY
-                 * until we closed everything on top of them. */
-                log_info("Unmounting %s.", m->path);
-                if (umount2(m->path, 0) == 0) {
-                        if (changed)
-                                *changed = true;
-                } else {
+                 * until we closed everything on top of them.
+                 *
+                 * In the event that an NFS or FUSE filesystem is
+                 * inaccessible, we never get here due to the previous
+                 * remount operation. */
+                if (umount_with_timeout(m, changed, log_error) < 0) {
+                        n_failed++;
                         if (log_error)
                                 log_warning_errno(errno, "Could not unmount %s: %m", m->path);
```

Comment:
umount_with-timeout() already logs in case of error, hence we can really drop this here, otherwise we'll see duplicate error messages, which is always confusing